cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/service/ServiceTask.java

```diff
@@ -171,6 +171,14 @@ public void setStartPort(int startPort) {
         this.startPort = startPort;
     }
 
+    public String getRequiredPermission() {
+        return this.properties.get("requiredPermission", String.class);
+    }
+
+    public void setRequiredPermission(String requiredPermission) {
+        this.properties.append("requiredPermission", requiredPermission);
+    }
+
```

Comment:
These methods shouldn't be in the ServiceTask-object. It seems pretty sensible, put this property is only part of the bridge module, not the cloud itself. When the bridge module is not loaded, this will be null. Furthermore, these two methods imply that this is a feature in the cloud itself, while it's not.

Reply:
removed them completely